inc/object.class.php

```diff
@@ -223,6 +223,18 @@ static function registerType() {
                array_push($ORDER_TYPES, $class);
             }
          }
+         if ($item->canBeReserved()) {
+            //Manage name used for sector
+            //See object.form.php L101
+            //it can be 'itemtype' name or 'family' name
+            if(!$name = PluginGenericobjectType::getFamilyNameByItemtype($class)){
```

Comment:
It will not work as expected.
```
var_dump(!$name = "test");
=> false
```

```suggestion
            if (($name = PluginGenericobjectType::getFamilyNameByItemtype($class)) === false) {
```